tests/certification/bindings/postgres/postgres_test.go

```diff
@@ -0,0 +1,241 @@
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for exec operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "exec", Metadata: metadata}
+		req.Metadata["sql"] = "INSERT INTO dapr_test_table (id, c1, ts) VALUES (1, 'demo', '2020-09-24T11:45:05Z07:00');"
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - exec")
+
+		return nil
+	}
+
+	testQuery := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for query operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "query", Metadata: metadata}
+		req.Metadata["sql"] = "SELECT * FROM dapr_test_table WHERE id = 1;"
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - query")
+
+		return nil
+	}
+
+	testClose := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for close operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "close", Metadata: metadata}
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - close")
+
+		return nil
+	}
+
+	createTable := func(ctx flow.Context) error {
+		db, err := sql.Open("postgres", dockerConnectionString)
+		assert.NoError(t, err)
+		_, err = db.Exec("CREATE TABLE dapr_test_table(id INT, c1 TEXT, ts TEXT);")
+		assert.NoError(t, err)
+		db.Close()
+		return nil
+	}
+
+	flow.New(t, "Run tests").
+		Step(dockercompose.Run("db", dockerComposeYAML)).
+		Step("wait for component to start", flow.Sleep(10*time.Second)).
+		Step("Creating table", createTable).
+		Step("wait for component to start", flow.Sleep(10*time.Second)).
+		Step(sidecar.Run("standardSidecar",
+			embedded.WithoutApp(),
+			embedded.WithDaprGRPCPort(grpcPort),
+			embedded.WithDaprHTTPPort(httpPort),
+			embedded.WithComponentsPath("./components/standard"),
+			componentRuntimeOptions(),
+		)).
+		Step("Run exec test", testExec).
+		Step("Run query test", testQuery).
+		Step("wait for DB operations to complete", flow.Sleep(10*time.Second)).
+		Step("Run close test", testClose).
+		Step("stop postgresql", dockercompose.Stop("db", dockerComposeYAML, "db")).
+		Step("wait for component to start", flow.Sleep(10*time.Second)).
+		Step("interrupt network", network.InterruptNetwork(30*time.Second, nil, nil, "5432")).
```

Comment:
You need to actually run an operation after the network interruption to test that the component recovers.